addons/l10n_in/models/res_partner.py

```diff
@@ -28,6 +28,7 @@ class ResPartner(models.Model):
     )
 
     display_pan_warning = fields.Boolean(string="Display pan warning", compute="_compute_display_pan_warning")
+    l10n_in_trade_name = fields.Char(string="Trade name")
```

Comment:
@ajjo-odoo @zepa-odoo It is making it cumbersome for the user to also have to fill that in on every partner/contact.

Reply:
> @ajjo-odoo @zepa-odoo It is making it cumbersome for the user to also have to fill that in on every partner/contact.

i have discord to you the use case and need of the field for every contact.
it concern with EDI.

Reply:
Hello, I agree with Josse, could you explain why it's needed (here) ? Thank you

Reply:
@jco-odoo @JulienVR 
Agree to remove trade name from every contact and just to keep it over res.company as per sepc usecase.
for every partner we will ask users to use reference field available for alias.
Thanks for the input.
@zepa-odoo 
We can adopt the changes accordingly.
Thanks

Reply:
Hello @ajjo-odoo, @jco-odoo and @JulienVR,

I have updated the code according to the new specifications. Please review the changes.

Reply:
@zepa-odoo 
Thanks tested and works fine.
thanks

cc- @jco-odoo @JulienVR 

https://drive.google.com/file/d/1jDuaod2Xq6ZTFhOh-S7VJK2sm16x8LBl/view?usp=sharing